src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,43 @@
+package duke.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * This Event class implements the characteristics of a task
+ * that is held on a specific date.
+ */
+public class Event extends Task {
+
+    protected LocalDate at;
```

Comment:
Maybe can be more specific? Similar point as Deadline.

Reply:
Good point, I will change this variable name too! Thanks! 😄